lib/enigma.rb

```diff
@@ -1,2 +1,29 @@
+require_relative './encryption'
+require_relative './decryption'
+require_relative './shifts'
+
 class Enigma
-end 
+  attr_reader :shifts
```

Comment:
Needed to add shifts as a method and an instance of a class in this file in order to stub it in the enigma test file. Might be a better way to do this, but for now it works!